README.md

```diff
@@ -19,18 +19,25 @@
 
 ## Status
 
-**ALPHA**
+**Completed**
+(last updated June 11th, 2015)
```

Comment:
this entire section should probably be rolled into the roadmap, with only the ALPHA status on the main README.